python/cuml/tsa/arima.pyx

```diff
@@ -976,7 +974,7 @@ class ARIMA(Base):
 
         return d_x_array.to_output("numpy")
 
-    @nvtx_range_wrap
+    @nvtx.annotate(message="cuml.tsa.arima.ARIMA._batched_transform")
```

Comment:
```suggestion
    @nvtx.annotate(message="tsa.arima.ARIMA._batched_transform")
```

If we use the domain we could do annotations without the `cuml.` so they would be a bit more concise.

Reply:
Done